ui/views/CredentialInfo.svelte

```diff
@@ -229,10 +231,19 @@
                 </header>
                 <ul>
                     {#each credentialInfo as object}
-                        <li>
-                            <p>{object.key}</p>
-                            <span>{object.value}</span>
-                        </li>
+                        {#if object.key[0] && object.key[0] === 'commitments'}
+                            {#each object.value as commitment}
+                                <li>
+                                    <p>{commitment.commitmentTitle} ({commitment.commitmentSupport})</p>
+                                    <span>{commitment.commitmentPercentage}</span>
+                                </li>
```

Comment:
@sobolev-alexey I'm happy to change this part if you want to adjust how commitment info is displayed

Reply:
This part is a little bit tricky. 
Please add a sub-condition to check which commitment type it is, then you can display the commitment using the following logic:

commitment === `FutureCommitment`
```
 <p>
       {commitment?.condition} {commitment?.percentage}% 
       THEN donate {commitment?.walletPercentage}% of my wallet balance TO support {commitment?.support}
</p>
```
commitment === `PresentCommitment`
```
  <p>I commit to {commitment?.support}</p>
```